pywebostv/controls.py

```diff
@@ -195,6 +202,7 @@ def list_audio_output_sources(self):
         "get_audio_output": {
             "uri": "ssap://audio/getSoundOutput",
             "validation": standard_validation,
```

Comment:
Here you'd probably pass `subscription_validation` instead of `standard_validation`. And remove the `"subscription_validation"` key on the next line. What do you think?

Reply:
I originally did that, the issue it that it needs both as non-subscribed calls still require the standard validation

Reply:
Makes sense, thank you!